app/src/ui/app.tsx

```diff
@@ -1100,6 +1102,19 @@ export class App extends React.Component<IAppProps, IAppState> {
     })
   }
 
+  /**
+   * Opens a browser to the issue creation page
+   * of the current GitHub repository.
+   */
+  private createIssueInRepositoryOnGitHub() {
+    const url = this.getCurrentRepositoryGitHubURL()
+
+    if (url) {
+      this.props.dispatcher.openInBrowser(`${url}/issues/new/choose`)
+      return
```

Comment:
Can we live without this explicit return?
```suggestion
```

Reply:
we can and we will!

(fwiw i copy pasta-ed this from other url-opening methods in `app.tsx`, so maybe we can clean those up next time we mess with them.)